.devcontainer/devcontainer.json

```diff
@@ -1,11 +1,26 @@
 {
   "name": "humble desktop",
   "dockerFile": "Dockerfile",
+  "features": {
+    "ghcr.io/devcontainers/features/desktop-lite:1": {}
+  },
+  "forwardPorts": [
+    6080
+  ],
+  "portsAttributes": {
+    "6080": {
+      "label": "desktop"
+    }
+  },
   //"image": "lcas.lincoln.ac.uk/lcas/devcontainer/engagement_detector:humble-dev",
   "runArgs": [
     // "--privileged",
     // "--network=host"
+    "--privileged",
```

Comment:
is 'priviledged` required?

Reply:
Not required to run the model, but to access a webcam from the container and run the demo.